packages/interface/src/token_cw20.rs

```diff
@@ -0,0 +1,51 @@
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::Binary;
+
+use crate::{
+    mailbox, router,
+    token::{TokenMode, TokenType},
+};
+
+#[cw_serde]
+pub enum ReceiveMsg {
+    // transfer to remote
+    TransferRemote { dest_domain: u32, recipient: Binary },
+}
+
+#[cw_serde]
+pub enum ExecuteMsg {
+    Router(router::RouterMsg),
+
+    // handle transfer remote
```

Comment:
chore; The comment should have this format
```suggestion
    /// handle transfer remote
```